include/dali/core/common.h

```diff
@@ -259,4 +259,6 @@ DLL_PUBLIC std::vector<std::string> string_split(const std::string &s, const cha
   if (1) \
   std::cerr << __FILE__ << ":" << __LINE__ << ": "
 
+#define DALI_DATALOADER_SEED 524287
```

Comment:
Why not a constexpr?

Reply:
I don't know, I hesitated between define and constexpr

Reply:
I will change to constexpr

Reply:
I don't have any strong opinion either. @jantonguirao - please share your thought.

Reply:
Ok let's wait for his opinion

Reply:
A constexpr is better in my opinion

Reply:
Done